Sources/PluginLibrary/plugin.pb.swift

```diff
@@ -118,13 +113,13 @@ public struct Google_Protobuf_Compiler_CodeGeneratorRequest: ProtobufGeneratedMe
 
   public func _protoc_generated_traverse(visitor: inout ProtobufVisitor) throws {
     if !fileToGenerate.isEmpty {
-      try visitor.visitRepeatedField(fieldType: ProtobufString.self, value: fileToGenerate, protoFieldNumber: 1, protoFieldName: "file_to_generate", jsonFieldName: "fileToGenerate", swiftFieldName: "fileToGenerate")
+      try visitor.visitRepeatedField(fieldType: ProtobufString.self, value: fileToGenerate, protoFieldNumber: 1)
     }
     if let v = _parameter {
-      try visitor.visitSingularField(fieldType: ProtobufString.self, value: v, protoFieldNumber: 2, protoFieldName: "parameter", jsonFieldName: "parameter", swiftFieldName: "parameter")
+      try visitor.visitSingularField(fieldType: ProtobufString.self, value: v, protoFieldNumber: 2)
     }
     if !protoFile.isEmpty {
-      try visitor.visitRepeatedMessageField(value: protoFile, protoFieldNumber: 15, protoFieldName: "proto_file", jsonFieldName: "protoFile", swiftFieldName: "protoFile")
+      try visitor.visitRepeatedMessageField(value: protoFile, protoFieldNumber: 15)
```

Comment:
I've had that in the back of my head for a while but was planning to do it in a separate PR, but I guess since I'm touching everything already... why not?